src/sysrepo.h

```diff
@@ -1156,9 +1160,13 @@ int sr_get_oper_changes(sr_session_ctx_t *session, const char *module_name, sr_d
  * for the module.
  *
  * By default the next highest order (lowest priority) is generated for any new session push oper data of a module.
+ * It is not supported to set the order:
+ * 1. if the session already has push operational data for the specified module.
+ * 2. for all modules if the session has push operational data for any module.
```

Comment:
Do not understand this, you cannot change the order of all the modules anymore no matter what, based on the `module_name` description (which is fine).